app/_components/_ui/main/christmas/snowy.tsx

```
@@ -0,0 +1,53 @@
+function snowAttributes() {
+  const startX = Math.random() * 100;
+  const endX = startX + (Math.random() * 20 - 10);
+  const scale = Math.max(Math.random(), 0.5);
+
+  return {
+    opacity: Math.random(),
+    keyframe: [{ transform: `translate(${startX}vw, 0) scale(${scale})` }, { transform: `translate(${endX}vw, 100vh) scale(${scale})` }],
+    option: {
+      duration: 15000 + Math.random() * 5000,
+      easing: 'linear',
+      iterations: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)